GenderPayGap.WebUI/Views/AdminDatabaseIntegrityChecks/DatabaseIntegrityChecks.cshtml

```diff
@@ -0,0 +1,195 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@{
+    ViewBag.Title = "Database integrity checks - Administration - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        var crumbs = new List<CrumbViewModel>
+        {
+            new CrumbViewModel
+            {
+                Text = "Admin",
+                Href = Url.Action("Home", "Admin")
+            },
+            new CrumbViewModel
+            {
+                Text = "Database integrity checks"
+            }
+        };
+    }
+
+    @(Html.GovUkBreadcrumbs(new BreadcrumbsViewModel
+    {
+        Crumbs = crumbs
+    }))
+}
+
+<main id="content" role="main">
+    <span class="govuk-caption-xl">Administration</span>
+    <h1 class="govuk-heading-xl">Database integrity checks</h1>
+
+    <div>
```

Comment:
It was intended to group similar results together with the related title. It also makes the file more readable because of the added indentation. Would a more semantic `<section>` tag be better?